db/migrate/20191009004932_create_bulk_downloads_table.rb

```diff
@@ -0,0 +1,18 @@
+class CreateBulkDownloadsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :bulk_downloads do |t|
+      t.text "params_json", comment: "JSON of the params for this bulk download"
```

Comment:
so `params_json` may be null?

Reply:
Yeah. For some types of downloads there are no params. We could require everything to have an empty json `{}`, or just interpret `null` as `{}`.